_posts/2017-02-22-Debezium-at-WePay.adoc

```diff
@@ -1,5 +1,6 @@
 ---
 layout: post
+canonical_url: https://wecode.wepay.com/posts/streaming-databases-in-realtime-with-mysql-debezium-kafka
```

Comment:
Cool; much nicer than before, putting the canonical link as an attribute right into files themselves 👍